hack/generate-manifest.sh

```diff
@@ -47,6 +47,7 @@ Generate a YAML manifest for Antrea using Helm and print it to stdout.
         --help, -h                    Print this message and exit
         --multicast                   Generates a manifest for multicast.
         --multicast-interfaces        Multicast interface names (default is empty)
+        --igmpquery-interval          Interval to send IGMP query, time unit is second (default is 125)
```

Comment:
This seems too specific. I think you can use `--extra-helm-values-file` or add a `--extra-helm-values` argument.

Reply:
removed this option, and use `--extra-helm-values-file`